example_tb/core/Makefile

```diff
@@ -198,6 +198,34 @@ custom-vsim-run-gui: vsim-all custom/hello_world.hex
 custom-vsim-run-gui: ALL_VSIM_FLAGS += "+firmware=custom/hello_world.hex"
 custom-vsim-run-gui: vsim-run-gui
 
+#Perfcnt test
```

Comment:
I really appreciate you added a test case for this. The test case will have to move to the core-v-verif repos instead though.